src/components/Header/Header.jsx

```diff
@@ -1,16 +1,44 @@
-import DestinationPopUp from "../DestinationPopUp/DestinationPopUp";
+import React from 'react'
+import styles from "./Header.module.css"
+import airbnbLogo from "../../assets/logo_airbnb.svg"
+import ButtonBeAHost from '../ButtonBeAHost/ButtonBeAHost'
+import LanguageSelector from '../LanguageSelector/LanguageSelector'
+import DestinationPopUp from '../DestinationPopUp/DestinationPopUp'
+import ToggleButtonsStaysExperiences from '../ToggleButtonsStaysExperiences/ToggleButtonsStaysExperiences'
+
 
 const Header = () => {
-  const handelRegionClick = (item) => {};
+    const handelRegionClick = (item) => {
+    }
 
-  return (
-    <div>
-      <DestinationPopUp
-        title="Search by region"
-        onClick={(e) => handelRegionClick(e)}
-      />
-    </div>
-  );
-};
+    return (
+        <div className={styles.headerContainer}>
+            <header className={styles.header}>
+                <div className={styles.logoDiv}>
+                    <a className={styles.airbnbHomeButton}
+                        href="/">
+                        <img src={airbnbLogo} className={styles.logoAirbnb} alt='The Logo of AirBnB' />
+                    </a>
+                </div>
+				<ToggleButtonsStaysExperiences />
+                {/* {<div className={styles.toggleButtonsStaysExperiences}>
```

Comment:
do we need this commented out code? Maybe better to get rid of it if its not needed...

Reply:
deleted